src/ipv6/ndpv6.ml

```diff
@@ -270,6 +274,10 @@ type na =
     na_target : Ipaddr.t;
     na_tlla : Macaddr.t option }
 
+type rdnss = 
+  { rdnss_lifetime: time option;
+    rdnss_addresses: Ipaddr.t }
```

Comment:
```suggestion
    rdnss_addresses: Ipaddr.V6.t list }
```

Reply:
`Ipaddr` is already aliased(?) to `Ipaddr.V6` in this file.